custom/icds_reports/models/aggregate.py

```diff
@@ -1688,3 +1690,51 @@ class Meta(object):
 
     _agg_helper_cls = AggServiceDeliveryReportHelper
     _agg_atomic = False
+
+
+class AggBiharAPIDemographics(models.Model, AggregateMixin):
```

Comment:
The agg name seems misleading here since it is one row per person. We typically use that name to signify that it is aggregated by location, whereas this is a monthly beneficiary table

Reply:
replaced the agg part.